docker-compose/docker-compose.yml

```diff
@@ -40,10 +41,11 @@ services:
     env_file:
       -  ./envs/common-blockscout.env
     environment:
-      ETHEREUM_JSONRPC_VARIANT: 'nethermind'
-      ETHEREUM_JSONRPC_HTTP_URL: http://staging.omni.network:8545
-      ETHEREUM_JSONRPC_TRACE_URL: http://staging.omni.network:8545
-      JSON_RPC: http://staging.omni.network:8545
+      PERMANENT_LIGHT_MODE_ENABLED: true
+      ETHEREUM_JSONRPC_VARIANT: 'geth'
```

Comment:
is it safe to change variant? i think there is some difference with things such as internal transactions indexing within different variants

also, this is not docker compose per env?

Reply:
geth is the variant we use in production and is the right one for our rpc. you're right the main differerence is what methods blockscout uses to trace internal txns.